README.md

```diff
@@ -14,15 +15,18 @@ The system provides the following functionalitites :
 - Periodic logging of the battery levels of robots into DB.
 
 ## Run the project locally
-RobotManagementSystem is a Spring Boot application built using Maven. You can build a jar file and run it from the command line (it should work with Java 17 or newer) : 
+
+RobotManagementSystem is a Spring Boot application built using Maven. You can build a jar file and run it from the command line (it should work with Java 17 or newer):
+

```

Comment:
This adheres to markdownlint rule [MD031](https://github.com/DavidAnson/markdownlint/blob/main/doc/md031.md).